src/fu-engine.c

```diff
@@ -1697,6 +1697,17 @@ fu_engine_get_report_metadata(FuEngine *self, GError **error)
 	if (tmp != NULL)
 		g_hash_table_insert(hash, g_strdup("HostBkc"), g_strdup(tmp));
 
+#ifdef HAVE_PASSIM
+#if PASSIM_CHECK_VERSION(0, 1, 6)
```

Comment:
If this isn't going to backport to 1.9.x then just make the dep 0.1.6 and you can lose an ifdef

Reply:
I was thinking of backporting it to 1.9.x, otherwise it's going to be a long wait indeed.